src/CloudFlareConfigInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudflareConfigInterface.
+ */
+
+namespace Drupal\cloudflare;
+
+/**
+ * Zone methods for CloudFlare.
+ */
+interface CloudFlareConfigInterface {
+  /**
+   * Determine the current Cloudflare ZoneId for the site.
+   *
+   * Most CloudFlare accounts have a single zone. It can be assumed as the
+   * default. If there is more than one the user needs to specify the zone.
+   * on the cloudflare config page.
+   *
+   * @return string
+   *   The id of the current zone.
+   */
+  public function getCurrentZoneId();
+
+  /**
+   * Gets the currently instantiated ZoneApi object for the site.
+   *
+   * @return \CloudFlarePhpSdk\ApiEndpoints\ZoneApi
+   *   Instance of ZoneApi for the site.
```

Comment:
I have no idea what this means.

"Instance of ZoneApi" -> ?

"[…] for the site" -> ?

Does the latter mean that this `ZoneApi` instance will only affect this specific site, i.e. this specific subdomain? I don't think that's possible?